README.md

```diff
@@ -69,17 +55,18 @@ plt.xlabel('Year')
 
 We invite you to go over the example notebooks in the `examples` directory.
 
-## Documentation
+# Documentation
 The documentation of the API and models is available [here](https://unit8co.github.io/u8timeseries/).
 
-## List of Features
+# Features
 Currently, the library contains the following features: 
 
 **Forecasting Models:** 
 * Exponential smoothing, 
 * ARIMA & auto-ARIMA,
 * Facebook Prophet,
-* Theta method, 
+* Theta method,
+* FFT (Fast Fourier Transform)
```

Comment:
This one is not actually released yet :)

Reply:
I know it will be very soon though, right @pennfranc ? :D